source/adios2/helper/adiosMpiHandshake.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Distributed under the OSI-approved Apache License, Version 2.0.  See
+ * accompanying file Copyright.txt for details.
+ *
+ * adiosMpiHandshake.h
+ */
+
+#ifndef ADIOS2_HELPER_ADIOSMPIHANDSHAKE_H_
+#define ADIOS2_HELPER_ADIOSMPIHANDSHAKE_H_
+
+#include "adios2/common/ADIOSConfig.h"
+#ifndef ADIOS2_HAVE_MPI
+#error "Do not include adiosCommMPI.h without ADIOS2_HAVE_MPI."
+#endif
```

Comment:
Please update the file name in the `#error` line to match this header.

Reply:
Done